index.html

```diff
@@ -97,6 +98,7 @@
     table.simple td {
         border-right: 1px solid #000;
     }
+    .issue[id^='issue-'] > *:not([role='heading']) {}
```

Comment:
This CSS rule has no properties. How does it "strip off ... the text taken from the issue description"? `document.querySelector(".issue[id^='issue-']")` also returns nothing, so the rule doesn't match anything anyway.